lncfg/protocol.go

```diff
@@ -29,6 +29,10 @@ type ProtocolOptions struct {
 	// opening or accepting channels having the script enforced commitment
 	// type for leased channel.
 	NoScriptEnforcedLease bool `long:"no-script-enforced-lease" description:"disable support for script enforced lease commitments"`
+
+	// OptionScidAlias should be set if we accept zero-conf channels and
+	// scid aliases.
+	OptionScidAlias bool `long:"option-scid-alias" description:"enable support for option_scid_alias and zero-conf channels"`
```

Comment:
> > As seems relevant to be able to enable the scid mapping, for existing channels
> 
> this would only be useful for private channels right?

Yes - the spec draft changed so if `announce_channel` != 0, then the `option_scid_alias` channel type shouldn't be set.